docs/snippets/collecting-metrics.md

```diff
@@ -47,27 +47,71 @@ aggregating timeseries metrics and alerting. It can also be used to scrape the O
     kubectl apply -f https://raw.githubusercontent.com/knative-extensions/monitoring/main/servicemonitor.yaml
     ```
 
-1. Grafana dashboards can be imported from the [`knative-extensions` repository](https://github.com/knative-extensions/monitoring/tree/main/grafana).
+### Access the Prometheus instance locally
+
+By default, the Prometheus instance is only exposed on a private service named `prometheus-operated`.
+
+To access the console in your web browser:
+
+1. Enter the command:
+
+    ```bash
+    kubectl port-forward -n default svc/prometheus-operated 9090
```

Comment:
I would use `kubectl port-forward -n prometheus svc/prometheus-kube-prometheus-prometheus 9090:9090` since that service is meant for [other reasons](https://stackoverflow.com/questions/69561623/what-does-prometheus-operated-service-do).

Reply:
Should I specify a new namespace (like _metrics_) for all `kubectl` commands or is better to maintain _default_?

Reply:
I think default is good enough for now, I am sure people can adjust it for their own purposes.